docs/terminal.md

```diff
@@ -2,7 +2,10 @@
 
 Open OnDemand provides access to a browser-based terminal. This terminal runs on a login node in the compute cluster, rather than a compute node, so the terminal environment should not be used directly for computation.
 
-Instead, the terminal should be used to queue batch or interactive jobs with slurm commands, or to perform simple, non-intensive tasks in your home directory.
+Instead, the terminal should be used to queue batch or interactive jobs with slurm commands like [sbatch](https://slurm.schedmd.com/sbatch.html), or to perform simple, non-intensive tasks in your home directory.
+
+!!! danger
```

Comment:
Wonder if we should use `warning` but otherwise the message looks good to me!

Reply:
I realize the `danger` admonition is a bit over the top. I'm certainly open to switching to `warning`.

Reply:
I tried out the `warning` admonition, but I think the `danger` version is a little more attention grabbing. Since ignoring it has consequences for everyone (user's process gets killed, other users locked out), I think the more attention grabbing version might be the way to go.